app/templates/identity.hbs

```diff
@@ -25,7 +26,14 @@
                     <i class="fa fa-spinner fa-spin"></i>
                 {{/if}}
                 Save
-            </button> 
+            </button>
```

Comment:
Can we please format the left-hand spacing in each line?

Reply:
fixed the  left hand spacing in each line